app.js

```diff
@@ -1,14 +1,34 @@
+'use strict';
 
+/* External modules */
 
-var express = require('express'),
-  config = require('./config/config'),
-  glob = require('glob');
+var fs = require('fs');
+var express = require('express');
+var winston = require('winston');
+
+/* Library modules */
+
+var CampaignInfoStorage = require('./lib/storage/sqlite/campaign-info-storage');
+
+/* Controllers */
+
+var campaigninfoById = require('./controllers/campaigns').campaigninfoById;
+
+/* App variables */
 
 var app = express();
+var config = JSON.parse(fs.readFileSync('config.json'));
```

Comment:
I don't really see any reason to have a dedicated config directory? Makes more sense to keep all config directives in a single file to me. This is more of a proposal than a conviction though. If you guys would rather break stuff out into separate files in a config directory, I'll get on board with it.